indico/modules/rb/util.py

```diff
@@ -329,3 +331,21 @@ def get_prebooking_collisions(reservation):
     from indico.modules.rb.models.reservation_occurrences import ReservationOccurrence
     valid_occurrences = reservation.occurrences.filter(ReservationOccurrence.is_valid).all()
     return ReservationOccurrence.find_overlapping_with(reservation.room, valid_occurrences, reservation.id).all()
+
+
+def check_impossible_repetition(data):
+    """
+    Check for broken repetition data
```

Comment:
```suggestion
    """Check for broken repetition data
```

https://peps.python.org/pep-0257/

> Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description.

> The summary line may be on the same line as the opening quotes or on the next line.

We generally do "on the same line" - the only (somewhat rare) exception are docstrings where we don't really have a title but just a body; in that case we don't start in the same line.

Reply:
I see - so would this be alright?

```suggestion
    """Check for broken repetition data
    
    ...detailed explanation goes here"""
```

Reply:
no, the title line usually ends with a `.` and the closing `"""` always goes on its own line (unless the whole docstring is single-line)

(my suggestion is already changing enough to match the style (nearly) all other docstrings in indico use...)